pkg/apis/monitoring/v1/types.go

```diff
@@ -145,7 +147,6 @@ type PrometheusSpec struct {
 	// data to a central location. Sharding is done on the content of the
 	// `__address__` target meta-label.
 	Shards *int32 `json:"shards,omitempty"`
-	// Name of Prometheus external label used to denote replica name.
```

Comment:
not sure why you've removed this line?

Reply:
If I did, did it accidentally. Will add back. Sorry for the inconvinience.